packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -55,17 +55,28 @@ export function ListViewItem(props) {
     onAction: onAction ? () => onAction(item.key) : undefined,
     shouldSelectOnPressUp: isListDraggable
   }, state, rowRef);
```

Comment:
useGridRow retains the same `node: item` so selection and actions use the row key still

Reply:
outdated now